Firebase/Auth/CHANGELOG.md

```diff
@@ -1,216 +1,216 @@
 # v6.2.1
-- Add new client error MISSING_CLIENT_IDENTIFIER. (#3341)
+- [added] Add new client error MISSING_CLIENT_IDENTIFIER. (#3341)
```

Comment:
The verb form here is all off. Does that matter? For example this one is "Added" in the proper release notes

https://firebase.google.com/support/release-notes/ios#authentication

Reply:
What's the difference between added and feature?

Reply:
None:

https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/make_release_notes.py#L88

Reply:
I'm considering putting together a script that checks for correct tense and use of [tags] in bulleted lists, but that will not be part of this change.